packages/validate/src/lib/dom/index.js

```diff
@@ -51,27 +51,29 @@ export const createErrorTextNode = (group, msg) => {
  * 
  */
 export const clearError = groupName => state => {
-    state.errorNodes[groupName].parentNode.removeChild(state.errorNodes[groupName]);
-    // errorNodes[groupName].parentNode.removeChild(errorNodes[groupName]);
     if (state.groups[groupName].serverErrorNode) {
+        state.groups[groupName].serverErrorNode.innerHTML = '';
```

Comment:
innerHTML = '' as the only error clearance method on serverErrorNodes to support SSR errors.